### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by filoozom** [babel/broccoli-babel-transpiler#207]
Hi,

Would it be possible to publish a new release with the `workerpool` update?
As far as I can tell, it (f0e3b0a39f1413aaff3245cfdc50c0a472920a52) didn't make it in `v7.8.0`.

I'm trying to track down an issue with `ember` and NodeJS 16, and it seems like it could be coming from here.

Thanks!

**Comment by patricklx** [comment]
same for me. its because worker.send now receives options as parameter, which should be an object. but its receiving an array in the older version of workerpool.

**Comment by gorner** [comment]
Our team recently ran into the same issue (with Ember v3.28 and Node v16, no issues on Node v14) - it's possible a new release would help fix this, but we are behind on some other dependencies so this may not be the only cause for those running into this issue. (in particular a couple of the addons we currently use depend on `ember-raf-scheduler` v0.1.0)

```
⠴ building... [Babel: ember-raf-scheduler > applyPatches]node:internal/validators:173
      throw new ERR_INVALID_ARG_TYPE(name, 'Object', value);
      ^

TypeError [ERR_INVALID_ARG_TYPE]: The "options" argument must be of type object. Received an instance of Array
    at ChildProcess.target.send (node:internal/child_process:734:7)
    at Array.forEach (<anonymous>)
    at dispatchQueuedRequests (<path>/node_modules/workerpool/lib/WorkerHandler.js:174:21)
    at ChildProcess.<anonymous> (<path>/node_modules/workerpool/lib/WorkerHandler.js:129:7)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess.emit (node:domain:489:12)
    at emit (node:internal/child_process:939:14)
    at processTicksAndRejections (node:internal/process/task_queues:84:21) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

**Comment by bertdeblock** [comment]
Was released in [v8.0.0-beta.1](https://github.com/babel/broccoli-babel-transpiler/blob/master/CHANGELOG.md#v800-beta1-2022-10-07), if I'm not mistaken.
